sdk/src/main/scala/com/horizen/account/websocket/data/Subscription.scala

```diff
@@ -0,0 +1,49 @@
+package com.horizen.account.websocket.data
+
+import com.horizen.account.receipt.Bloom
+import com.horizen.evm.interop.EvmLog
+import com.horizen.evm.utils.Hash
+import com.horizen.utils.BytesUtils
+import org.web3j.utils.Numeric
+
+import jakarta.websocket.Session
+
+trait BaseSubscription {
+  def session: Session
+  def subscriptionId: String
+}
+
+case class Subscription(session: Session, subscriptionId: String) extends BaseSubscription {}
+case class SubscriptionWithFilter(session: Session, subscriptionId: String,
+                                  address: Option[Array[String]],
```

Comment:
```suggestion
                                  address: Option[Address],
```

Reply:
Do you mean Array[String] -> Array[Address]?

Reply:
Done